recipe/meta.yaml

```diff
@@ -24,78 +24,76 @@ requirements:
     - python-build
     - meson-python ==0.13.1
     - meson ==1.2.1
-    - cython >=0.29.33,<3
+    - cython >=3.0.5
     - numpy {{ numpy }}
     - versioneer
-    - tomli # [py<311]
     - pip
   run:
     - python
     # Base Dependencies
     - {{ pin_compatible('numpy') }}
     - python-dateutil >=2.8.2
     - pytz >=2020.1
-    - python-tzdata >=2022.1
+    - python-tzdata >=2022.7
     # Recommended dependencies to achieve large speedups
     # see https://pandas.pydata.org/pandas-docs/version/2.1.1/getting_started/install.html#performance-dependencies-recommended
-    - numexpr >=2.8.0
-    - bottleneck >=1.3.4
+    - numexpr >=2.8.4
+    - bottleneck >=1.3.6
   run_constrained:
     # Optional dependencies
     # see https://pandas.pydata.org/pandas-docs/version/2.1.1/getting_started/install.html#optional-dependencies
     # and https://github.com/pandas-dev/pandas/blob/v2.1.1/pyproject.toml
     # performance
```

Comment:
Were we intentionally leaving [these](https://github.com/pandas-dev/pandas/blob/v2.2.1/pyproject.toml#L90-L91) dependencies out? I noticed they might need to be built as well, so I was wondering if that effort was already deemed unnecessary, even though at quick glance they seem easy to add.

Reply:
This was a failed experiment on my part, so I put it back the way it was.